rspace/src/main/scala/coop/rchain/rspace/HotStore.scala

```diff
@@ -60,63 +54,60 @@ final case class Cache[C, P, A, K](
 }
 
 private class InMemHotStore[F[_]: Concurrent, C, P, A, K](
-    implicit S: Ref[F, Cache[C, P, A, K]],
+    cacheRef: Ref[F, Cache[C, P, A, K]],
```

Comment:
I still hope to put hotstore refactor <https://github.com/rchain/rchain/pull/3271> at some point. Just a reminder